examples/xed-examples-util.c

```diff
@@ -219,7 +219,7 @@ xed_int64_t xed_atoi_general(char* buf, int mul) {
         {
             p++;
         }
-        while(*p && isdigit(*p))
+        while(*p && isdigit((unsigned char)*p))
```

Comment:
ditto. from NetBSD isdigit(3) man page.
> CAVEATS
>      The argument to isdigit() must be EOF or representable as an unsigned
>      char; otherwise, the behavior is undefined.  See the CAVEATS section of
>      ctype(3) for more details.